src/lib.rs

```diff
@@ -1,19 +1,20 @@
+use std::collections::HashMap;
 use std::ffi::OsString;
 use std::iter::FromIterator;
 
 /// Structure to deal with environment variables
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct Environment {
     /// Customized environment variables
-    vars: Vec<(OsString, OsString)>,
+    vars: HashMap<OsString, OsString>,
     /// Define if the structure must inherit
     inherit: bool,
 }
 
 impl Default for Environment {
     fn default() -> Self {
         Self {
-            vars: vec![],
+            vars: HashMap::new(),
             inherit: false,
```

Comment:
Is this flag needed anymore?

Reply:
I leaved it for now but it will be removed next release.